src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -294,9 +294,9 @@ def load_arguments(self, _):
         c.argument('assign_kubelet_identity', validator=validate_assign_kubelet_identity)
         c.argument('enable_aad', action='store_true')
         c.argument('enable_azure_rbac', action='store_true')
-        c.argument('aad_client_app_id')
-        c.argument('aad_server_app_id')
-        c.argument('aad_server_app_secret')
+        c.argument('aad_client_app_id', deprecate_info=c.deprecate(target='--aad-client-app-id', hide=True))
```

Comment:
`target` could be ignored for all these 3 options as there's no special option form for these options.

Reply:
Not sure if it's a good idea to hide these options directly. Maybe hide them after several official cli release?

Reply:
> `target` could be ignored for all these 3 options as there's no special option form for these options.

See https://github.com/Azure/azure-cli/issues/24647

Reply:
got it

Reply:
> Not sure if it's a good idea to hide these options directly. Maybe hide them after several official cli release?

"hide=True" will hide these properties in the help message if the customer runs `az aks create -h`. The customer still can use them.

If it's preferred to add "hide=True" after several official cli release, I'm fine with it.

Reply:
After re-thinking, I'll add "hide=True" back because other parameters have this option too. To keep aligned, will add "hide=True" back.